src/services/collection.service.js

```diff
@@ -49,6 +49,26 @@ async function deleteFromCollection(collectionId, productId) {
   return deletedProduct;
 }
 
+async function findCollection(userId, collectionId, { offset, limit }) {
```

Comment:
There is a fault in calculating TotalPages. Look into that.

Reply:
yes  I just resolved them